.github/workflows/checks.yml

```
@@ -18,6 +18,11 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - run: wget https://github.com/editorconfig-checker/editorconfig-checker/releases/download/v3.2.0/ec-linux-amd64.tar.gz
+      - name: Verify checksum
+        run: |
+          if [ "$(sha256sum ec-linux-amd64.tar.gz  | cut -d' ' -f1)" != "5e6a63097904be33c8d18e960f54fd8f60ada5464fe0056cd3dbbd0678584d15" ]; then
+            echo "Checksum doesn't match! Maybe someone added malware to the release?"
```

This is kinda important :D ...

```suggestion
            echo "Checksum doesn't match! Maybe someone added malware to the release?"
            exit 1
```